src/email/mailer/mailer.js

```diff
@@ -15,47 +15,45 @@ const sendRawMail = (transporter, data) =>
 
 // prefix is shitty and used only by the tests, I'm tired by this and dont want to find for another way to handle this
 // enjoy yourself
-const sendCertificate = transporter => async ({
-  to,
-  certificate,
-  prefix = ""
-}) => {
-  const { html, text, subject } = messageTemplate(certificate);
-  const content = JSON.stringify(certificate);
+const sendCertificate =
+  (transporter) =>
+  async ({ to, certificate, prefix = "" }) => {
+    const { html, text, subject } = messageTemplate(certificate);
+    const content = JSON.stringify(certificate);
 
-  // eslint-disable-next-line no-console
-  console.log("Email:", to);
-  // eslint-disable-next-line no-console
-  console.log("Certificate:", content);
+    // eslint-disable-next-line no-console
+    console.log("Email:", to);
+    // eslint-disable-next-line no-console
+    console.log("Certificate:", content);
 
-  return sendRawMail(transporter, {
-    to,
-    from: FROM_ADDRESS,
-    subject,
-    html,
-    text,
-    attachments: [
-      {
-        filename: "logo.png",
-        path: `${prefix}static/logo.png`,
-        cid: "logo"
-      },
-      {
-        filename: "certificate.png",
-        path: `${prefix}static/certificate.png`,
-        cid: "certificate"
-      },
-      {
-        filename: "dropzone.png",
-        path: `${prefix}static/dropzone.png`,
-        cid: "dropzone"
-      },
-      {
-        filename: FILE_NAME,
-        content
-      }
-    ]
-  });
-};
+    return sendRawMail(transporter, {
+      to,
+      from: FROM_ADDRESS,
+      subject,
+      html,
+      text,
+      attachments: [
+        {
+          filename: "logo.png",
+          path: `${prefix}static/logo.png`,
+          cid: "logo"
+        },
+        {
+          filename: "certificate.png",
+          path: `${prefix}static/certificate.png`,
+          cid: "certificate"
+        },
+        {
+          filename: "dropzone.png",
+          path: `${prefix}static/dropzone.png`,
+          cid: "dropzone"
+        },
+        {
+          filename: FILE_NAME,
+          content
+        }
+      ]
+    });
+  };
 
 module.exports = sendCertificate;
```

Comment:
Do we also have to conform to ESM exporting here?

```suggestion
export sendCertificate;
```

Reply:
Let me check as these files (not directly related to the index.js or is a handler) did not throw any errors

Reply:
Just checked, we dont have to conform to ESM exporting here. Mainly the serverless handlers are affected by this change.